testbed/tests/test_images.py

```diff
@@ -22,16 +22,6 @@ async def test_local_image(app):
     assert image.height == 72
 
 
-async def test_raw_image(app):
```

Comment:
Presumably we still need to *test* this case - it *was* valid previously, and clearly has different semantics on some platforms.

Reply:
I have added it back.